src/main/groovy/edu/oregonstate/mist/locations/mapper/LocationMapper.groovy

```diff
@@ -188,4 +196,17 @@ class LocationMapper  {
             )
         }
     }
+
+    /**
+     * Return boolean giRestroomLimit based on contents of string
+     */
+    private Boolean getGiRestroomLimit(String restroomLimitString) {
+        if (restroomLimitString && restroomLimitString.trim().length() > 0 ) {
+            return true
+        } else if (restroomLimitString && restroomLimitString.trim().length() == 0 ) {
```

Comment:
If you use a couple of groovy shortcuts this method could be simplified to something like:
if (restroom == null) {
return null
} 

restroomLimitString?.trim()

In groovy, empty or null strings evaluate to false: http://docs.smartthings.com/en/latest/getting-started/groovy-basics.html#groovy-truth . Let's also add a unit test for this method